src/Preview.tsx

```diff
@@ -201,6 +202,31 @@ const Preview: React.FC<PreviewProps> = props => {
     setLastWheelZoomDirection({ wheelDirection });
   };
 
+  const onKeyDown = useCallback(
+    (event: KeyboardEvent) => {
+      if (!visible || !showLeftOrRightSwitches) return;
+
+      event.preventDefault();
+      if (event.key === 'ArrowLeft' || event.keyCode === KeyCode.LEFT) {
```

Comment:
Use `rc-util/lib/KeyCode` instead

Reply:
Because this attribute is marked as obsolete on MDN, I use the recommended key attribute when judging, and also use keyCode (rc-util/lib/KeyCode constant) for compatibility.

https://developer.mozilla.org/zh-CN/docs/Web/API/KeyboardEvent

Reply:
yes. Use legacy `rc/KeyCode` that we can do refactor in future on all rc components.

Reply:
Deleted, please review again. @zombieJ